### ISSUE TITLE
pbss query gives unexpected results

### ISSUE THREAD

**Issue by du5** [bnb-chain/bsc#2197]
```golang 
var (
	bn *big.Int = big.NewInt(35_300_000)
)
totalSupply, err := ec.CallContract(context.Background(), ethereum.CallMsg{
	From: common.HexToAddress("0x0"),
	To:   &nft,
	Data: crypto.Keccak256([]byte("totalSupply()"))[:4],
}, bn)
if err != nil {
	panic(err)
}
```

```bash
geth --datadir=fastpbss --history.transactions=0 --tries-verify-mode=local --db.engine=pebble --maxpeers=256 --syncmode=full --ipcpath=/opt/ipc.ipc --port=30311 --discovery.port=30311 --config=config_pbss.toml --state.scheme=path --db.engine=pebble
```

When I use Hash-Base database it does it normally, After I completed the conversion via `geth db hbss-to-pbss --syncmode=full`, this query returned err: `missing trie node 32abcbadcd4a84f51d4ac75c17c98621cc0ae1dca32b4451aca858209d612bf4 (path ) state 0x32abcbadcd4a84f51d4 ac75c17c98621cc0ae1dca32b4451aca858209d612bf4 is not available`, I have not prune the node

btw, pbss database seems unable to run the prune command

```bash
➜  /opt du -h geth.pbss
9.0M	geth.pbss/geth/nodes
1.6T	geth.pbss/geth/chaindata/ancient/chain
4.0G	geth.pbss/geth/chaindata/ancient/state
1.6T	geth.pbss/geth/chaindata/ancient
2.4T	geth.pbss/geth/chaindata
2.4T	geth.pbss/geth
0	geth.pbss/keystore
2.4T	geth.pbss
➜  /opt 


➜  /opt geth snapshot prune-state --datadir /opt/geth.pbss
INFO [02-01|15:22:47.969] Maximum peer count                       ETH=50 total=50
INFO [02-01|15:22:47.972] Set global gas cap                       cap=50,000,000
INFO [02-01|15:22:47.972] Initializing the KZG library             backend=gokzg
INFO [02-01|15:22:47.991] Using pebble as the backing database
INFO [02-01|15:22:47.991] Allocated cache and file handles         database=/opt/geth.pbss/geth/chaindata cache=409.00MiB handles=524,288 "memory table"=102.25MiB
INFO [02-01|15:22:48.255] Opened ancient database                  database=/opt/geth.pbss/geth/chaindata/ancient/chain readonly=false frozen=35,666,711
CRIT [02-01|15:22:48.257] Offline pruning is not required for path scheme

➜  /opt geth snapshot prune-block --datadir /opt/geth.pbss --datadir.ancient /opt/geth.pbss/geth/geth/chaindata/ancient                       
INFO [02-01|15:38:26.231] Maximum peer count                       ETH=50 total=50
INFO [02-01|15:38:26.234] Set global gas cap                       cap=50,000,000
INFO [02-01|15:38:26.234] Initializing the KZG library             backend=gokzg
INFO [02-01|15:38:26.255] Using pebble as the backing database 
INFO [02-01|15:38:26.255] Allocated cache and file handles         database=/opt/geth.pbss/geth/chaindata cache=409.00MiB handles=524,288 "memory table"=102.25MiB
INFO [02-01|15:38:26.449] Opened ancient database                  database=/opt/geth.pbss/geth/geth/chaindata/ancient/chain readonly=false frozen=0
Chain metadata
  databaseVersion: 8 (0x8)
  headBlockHash: 0x06f0230d05a68cfdd8c07d5ff36a3b9bc1a9c7086cf8642f2eb47c4db05fef3c
  headFastBlockHash: 0x06f0230d05a68cfdd8c07d5ff36a3b9bc1a9c7086cf8642f2eb47c4db05fef3c
  headHeaderHash: 0x06f0230d05a68cfdd8c07d5ff36a3b9bc1a9c7086cf8642f2eb47c4db05fef3c
  lastPivotNumber: 35629523 (0x21fa9d3)
  len(snapshotSyncStatus): 287 bytes
  snapshotDisabled: false
  snapshotJournal: 7814581 bytes
  snapshotRecoveryNumber: <nil>
  snapshotRoot: 0xf046eaf0f89e41429558f30d6bfaaf367739149f88dcd2c4413a8d11036edf81
  txIndexTail: 0 (0x0)
  fastTxLookupLimit: 2350000 (0x23dbb0)


Fatal: Could not open database: ancient chain segments already extracted, please set --datadir.ancient to the correct path
```

**Comment by joeylichang** [comment]
1. pbss schema only support latest 128 blocks' world states, so 35_300_000 maybe old.
2. pbss schema do not need prune state, because it only one block world state in disk. 
3. ablout `Fatal: Could not open database` error, please use the newest version, it is solved.

**Comment by du5** [comment]
1, bad news
2, got it 
3, Confirmed to be available in v1.3.8
